jekyll/_cci2/executor-types.md

```diff
@@ -59,7 +59,8 @@ The following example specifies the default `machine` image.
 ```YAML
 jobs:
   build:
-    machine: true
+    machine:
+      enabled: true
```

Comment:
Pretty sure you don't need the "enabled" key here.

Reply:
oh tru !! that makes sense, so it's more like the docker syntax